GliaWidgets/ViewController/Survey/SurveyViewController.swift

```diff
@@ -0,0 +1,88 @@
+import UIKit
+
+extension Survey {
+    final class ViewController: UIViewController {
+
+        struct Props {
+            let header: String
+            var questionsProps: [QuestionPropsProtocol]
+            var submit: (Props) -> Void
+            var cancell: () -> Void
+
+            init(
+                header: String = "",
+                props: [QuestionPropsProtocol] = [],
+                submit: @escaping (Props) -> Void = { _ in },
+                cancel: @escaping () -> Void = {}
+            ) {
+                self.header = header
+                self.questionsProps = props
+                self.submit = submit
+                self.cancell = cancel
+            }
+        }
+
+        var props: Props {
+            didSet { render() }
+        }
+
+        required init?(coder: NSCoder) { preconditionFailure() }
```

Comment:
we usually mark the function `@available(*, unavailable)` so it could not be called and would throw compiler error if used

Reply:
fixed - 4c5f7f55d7a12723db9c4721f36e842cd93ad69f